src/grafana_dashboards/MongoDB_ReplSet_Summary.json

```diff
@@ -2590,7 +2593,7 @@
                     "uid": "${prometheusds}"
                 },
                 "definition": "label_values(up{juju_model=\"$juju_model\",juju_model_uuid=\"$juju_model_uuid\",juju_application=\"$juju_application\"},juju_unit)",
-                "hide": 0,
+                "hide": 2,
```

Comment:
is this needed

Reply:
Not needed but useful: It hides the `juju_unit` selector which is not used anymore in this dashboard